src/test/java/mil/dds/anet/test/resources/PositionResourceTest.java

```diff
@@ -45,6 +48,12 @@
       + " } organization { " + ORGANIZATION_FIELDS + " }";
   private static final String PREVIOUS_PEOPLE_FIELDS =
       " previousPeople { startTime endTime position { uuid } person { uuid name rank role } }";
+  private static PositionHistoryDao positionHistoryDao;
+
+  @BeforeAll
+  public static void setUpClass() {
+    positionHistoryDao = new PositionHistoryDao();
+  }
```

Comment:
Why not just `private static final PositionHistoryDao positionHistoryDao = new PositionHistoryDao();` ?